build.gradle

```diff
@@ -30,7 +30,6 @@ allprojects {
         jcenter()
         mavenCentral()
         maven { url "https://repo1.maven.org/maven2" }
```

Comment:
Remove this as well. Why do we need this?

Reply:
I didn't know why it was been used, not used anywhere. I tested the app working fine with removing this dependency.